src/app/multi-editor/multi-editor.component.ts

```diff
@@ -75,9 +76,14 @@ export class MultiEditorComponent implements OnInit {
     this.previewedActions = this.userActions;
     this.queryService.previewActions(this.userActions, this.lastSearchedQuery, this.currentPage, this.pageSize)
       .then((res) => {
```

Comment:
this endpoint should return an typed object (needs an `interface`), so that we don't have `['property']` access.